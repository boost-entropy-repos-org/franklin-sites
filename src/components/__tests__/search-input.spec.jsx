import React from 'react';
import { render } from '@testing-library/react';
import SearchInput from '../search-input';

describe('SearchInput', () => {
  test('should render', () => {
    const { asFragment } = render(<SearchInput placeholder="This is the initial value" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
