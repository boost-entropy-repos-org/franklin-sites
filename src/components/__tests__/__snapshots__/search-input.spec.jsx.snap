// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput should render 1`] = `
<DocumentFragment>
  <span
    class="search-input"
  >
    <input
      placeholder="This is the initial value"
      props="[object Object]"
      type="text"
    />
    <test-file-stub
      classname="search-input__icon"
      height="14"
      width="14"
    />
  </span>
</DocumentFragment>
`;
